modules/ROOT/pages/to-download-and-install-studio-lx.adoc

```diff
@@ -3,27 +3,39 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-To support some of Studio Themes in Linux, it is recommended to have GTK version 2 installed.
+== Prerequisites
 
-* Before downloading and launching Anypoint Studio, make sure you have the Java SE JDK 8 installed.
+* Ensure that you have the Java SE JDK 8 installed and configured.
+* To support some of Studio Themes in Linux, it is recommended to have GTK version 2 installed.
+* Download the Anypoint Studio installation file from the https://www.mulesoft.com/lp/dl/studio[Anypoint Studio Download Site].
+
+[NOTE]
+Anypoint Studio 7.x only supports Mule 4.x projects since the structure of the project, export format, XML and scripting language are different.
 
-To install Anypoint Studio:
 
-. Download Anypoint Studio from the https://www.mulesoft.com/lp/dl/studio[Anypoint Studio Download Site] and extract it.
+== Installation
+
 . Open the extracted Anypoint Studio.
 . Click OK to accept the default workspace. +
-When setting a custom path to your workspace, keep in mind that Anypoint Studio does not expand the `~` tilde used in Unix/Linux systems. It is recommended to use the absolute path when defining your workspace.
-+
-[NOTE]
---
-You cannot reuse a workspace used by Anypoint Studio 6.x or below. +
-If you choose to use an existing workspace, this must have been created using Anypoint Studio 7.0 or above.
---
+Use an absolute path when defining your workspace because Anypoint Studio does not expand the `~` tilde used in Unix/Linux systems. You cannot reuse a workspace used by Anypoint Studio 6.x or below. If you choose to use an existing workspace, this must have been created using Anypoint Studio 7.0 or above.
+
 
-Anypoint Studio displays a welcome message when it loads for the first time. +
-Click Get Started to start using Anypoint Studio.
+== Troubleshooting
 
+If Studio fails to load with a *An Error Has Occured* message, you must configure Anypoint Studio to point to your JDK installation as the default VM:
```

Comment:
Does Studio fail or does your project fail?
Consider referring to Studio in the first case as Anypoint Studio and in the second as just Studio, instead of the reverse (full name first, short name second).
If Studio is what fails, then you don't even have to repeat; you can use "you must reconfigure it to point to your...."